packages/safe-ds-eda/types/state.d.ts

```diff
@@ -77,26 +77,25 @@ export interface Profiling {
 
 export interface ProfilingDetailBase {
```

Comment:
What is the meaning of `bold`? The other words make more sense as an interpretation.